app/controllers/discourse_activity_pub/post_controller.rb

```diff
@@ -32,7 +32,6 @@ def unschedule
     protected
 
     def ensure_first_post
-      return render_post_error("first_post_not_enabled", 403) unless @post.activity_pub_first_post
       render_post_error("not_first_post", 422) unless @post.activity_pub_is_first_post?
```

Comment:
```
      render_post_error("not_first_post", 422) unless @post.activity_pub_is_first_post?
```

I'm not sure this PR is opening up publication scheduling to all posts, looks like we still check for `@post.activity_pub_is_first_post?`. 

Or I guess, it opens up scheduling for the first post for categories with Full Topic publication configured?